src/model_config_utils.cc

```diff
@@ -1620,6 +1620,38 @@ ValidateModelConfig(
             " cache.");
   }
 
+  // If model_metric is specified, validate it.
+  if (config.has_model_metrics()) {
+#ifdef TRITON_ENABLE_METRICS
+    for (const auto& metric_control : config.model_metrics().metric_control()) {
```

Comment:
Assuming this section will continue to grow, would be good to break out to a helper function ValidateModelMetricsConfig - but this is fine inline for now.

Reply:
Refactored to a helper function.